flow/scripts/synth.tcl

```diff
@@ -1,3 +1,5 @@
+set ::env(VERILOG_FILES) $::env(RESULTS_DIR)/1_synth.rtlil
```

Comment:
I believe this will fail with a cached netlist.

Reply:
Good poin, though I dont see how.

Will test manually report/fix as needed

Reply:
I don't think that file will exists though I haven't tried it.